static/app/views/settings/components/settingsBreadcrumb/index.tsx

```diff
@@ -40,17 +40,14 @@ function SettingsBreadcrumb({className, routes, params}: Props) {
         }
         const pathTitle = pathMap[getRouteStringFromRoutes(routes.slice(0, i + 1))];
         const isLast = i === lastRouteIndex;
-        // @ts-expect-error TS(7053): Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
-        const createMenu = MENUS[route.name];
-        const Menu = typeof createMenu === 'function' && createMenu;
+        const Menu = MENUS[route.name];
         const hasMenu = !!Menu;
 
         if (hasMenu) {
           return (
             <Menu
               key={`${route.name}:${route.path}`}
               routes={routes}
-              params={params}
```

Comment:
`params` had the wrong type so I just defaulted to `useParams` in the Menu itself.